packages/snap-preact/src/Snap.test.tsx

```diff
@@ -383,7 +383,7 @@ describe('Snap Preact', () => {
 				},
 			};
 
-			const client = new MockClient(searchConfig.client!.globals);
+			const client = new MockClient(searchConfig.client!.globals as TrackerGlobals);
```

Comment:
For all of the MockClient changes here - can we switch to typing as `ClientGlobals` instead of `TrackerGlobals`?  They are similar but slightly different.